paper-typeahead-input.html

```diff
@@ -313,7 +340,7 @@
         }
       });
       var input = Polymer.dom(this.root).querySelector('paper-input');
-      input.$.input.autocomplete = 'address-level4';
+      input.$.input.autocomplete = this.autocomplete;
```

Comment:
Without autocomplete="Off" I have suggest(browser) on
suggest(paper-typeahead-input). And browser suggest blocks
paper-typeahead-input suggesting. Maybe you know another way to resolve
this issue?
[image: Inline images 1]

On 13 July 2016 at 23:31, Tianxiang Zhang <notifications@github.com> wrote:

> In paper-typeahead-input.html
> <https://github.com/cheonhyangzhang/paper-typeahead-input/pull/30#discussion_r70702583>
> :
>
> > @@ -313,7 +340,7 @@
> >          }
> >        });
> >        var input = Polymer.dom(this.root).querySelector('paper-input');
> > -      input.$.input.autocomplete = 'address-level4';
> > +      input.$.input.autocomplete = this.autocomplete;
>
> Do you have any use case you think it's necessary to assign this
> autocomplete value of the input?
>
> This line of code is to fix the Chrome Autofilling problem.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cheonhyangzhang/paper-typeahead-input/pull/30/files/dfdc34b0472906d1f9c7005d4442b0162ab8c4f8#r70702583>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AG0SnOjkUkLbKRf2A1xmursYW3wXOD8nks5qVUsYgaJpZM4JKWUL>
> .
>



-- 
C уважением, Пличко Павел